.github/workflows/docs_deploy.yml

```
@@ -0,0 +1,266 @@
+      # We want to re-use it later for the build, hence 'tox run --notest' instead of 'tox devenv'.
+      - name: Prepare Python environment
+        run: tox run -e docs --notest
+
+      # The reason to use the custom script rather than letting 'nbsphinx' do its thing normally
+      # within the Sphinx build is so that the execution process is the same as in the test CI.
+      - name: Execute tutorials in place
+        run: .tox/docs/bin/python tools/execute_tutorials.py docs/tutorials
+        env:
+          QISKIT_CELL_TIMEOUT: "300"
+
+      - name: Build documentation
+        # We can skip re-installing the package, since we just did it a couple of steps ago.
+        run: tox run -e docs --skip-pkg-install
+        env:
+          QISKIT_ENABLE_ANALYTICS: "true"
+          # We've already built them.
+          QISKIT_DOCS_BUILD_TUTORIALS: "never"
+
+      - name: Build translatable strings
+        run: tox -e gettext
+        env:
+          # We've already built them.
+          QISKIT_DOCS_BUILD_TUTORIALS: "never"
+
+      - name: Store built documentation artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: qiskit-docs
+          path: |
+            ./docs/_build/html/*
+            !**/.doctrees
+            !**/.buildinfo
+          if-no-files-found: error
+
+      - name: Store translatable strings artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: qiskit-translatables
+          path: ./docs/locale/en/*
+          if-no-files-found: error
+
+  deploy:
+    if: github.event_name != 'workflow_dispatch' || inputs.do_deployment
+    name: Deploy to qiskit.org
+    needs: [build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: qiskit
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: qiskit-docs
+          path: deploy
+
+      - id: choose
+        name: Choose deployment location(s)
+        run: |
+          set -e
+          declare -a prefixes
+          case ${{ github.event_name }} in
+            push)
+              case ${{ github.ref_type }} in
+                branch)
+                  if [[ "$GITHUB_REF_NAME" != "main" ]]; then
+                    echo "Push to unhandled branch '$GITHUB_REF_NAME'" >&2
+                    exit 1
+                  fi
+
+                  prefixes+=( "dev" )
+                  ;;
+                tag)
+                  tag=$GITHUB_REF_NAME
+                  echo "Full tag: ${tag}"
+                  IFS=. read -ra version <<< "$tag"
+                  minor_version="${version[0]}.${version[1]}"
+                  echo "Minor version: ${minor_version}"
+                  prefixes+=( "stable/${minor_version}" )
+                  if [[ "$tag" == "$LATEST_TAG" ]]; then
+                    # Deploy to the root as well.
+                    prefixes+=( "" )
+                  fi
```

This will force us to push to the correct location on tags from the stable/0.25 branch.
```suggestion
                  prefixes+=( "stable/0.44" )
                  prefixes+=( "" )
```